### DISCUSSION TITLE
Data polling / refresh rate

### DISCUSSION THREAD

**Discussion by max10019679** [CJNE/ha-myenergi#481] [Q&A]
Hi I have the below automation set up to poll myenergi for data every 5 seconds. But it doesn’t seem to work. At best it polls every 10 seconds but is inconsistent. Anyone managed to successfully poll the devices regularly and how?
![IMG_0806](https://github.com/CJNE/ha-myenergi/assets/157296649/bee42323-3cef-497a-bae5-225766435d17)

**Comment by G6EJD** [comment]
But that’s getting on for 43200 API calls a day, that’s madness

**Comment by LeiChat** [comment]
Do myenergi capture the data from the devices and share via the API at that high a resolution?
